web/ui/react-app/src/PanelList.tsx

```diff
@@ -17,7 +18,7 @@ interface PanelListState {
 
 class PanelList extends Component<any, PanelListState> {
   private key: number = 0;
-
+  private initialMetricsNames: string[] = [];
```

Comment:
I think it'd just be "metric names" rather than "metrics names".

Reply:
thanks, will fix the typo